Makefile

```diff
@@ -60,5 +60,8 @@ podman_cleanup:
 bell:
 	@echo "Done!" $$'\a'
 
-assisted_deployment:
+assisted_deployment_complete:
```

Comment:
There is a possibility that changing this target name will break some existing workflow. I would suggest leaving it unchanged.

Reply:
Agreed!  Let's perhaps keep the current name (and deprecated it)

Reply:
Yeah, I just kept the current one. We never know what may explode...